test/Base_Tests/src/Data/Polyglot_Spec.enso

```diff
@@ -34,6 +34,14 @@ add_specs suite_builder = suite_builder.group "Polyglot" group_builder->
     group_builder.specify "access Integer field from Polyglot object" <|
         js_meaning.meaning . should_equal 42
 
+    group_builder.specify "Execute JavaScript function" <|
```

Comment:
would be nice to also see what happens if I try to specify more or less arguments to the function. Can you do, e.g.,
```
curried = js_plus 3
curried 5 . should_equal 8
```
?

Reply:
Invoking:
```ruby
curried = js_plus 3

main =
    curried

foreign js js_plus = """
    return (a, b) => a + b;
```
yields `NaN` as `3 + undefined` is `NaN`.

Reply:
I'd expect it to behave the same way as if `js_plus` was an enso function. I.e. `main` should return a function

Reply:
Tested in fe14b8d6d9

> I'd expect it to behave the same way as if js_plus was an enso function

You might, but there is **no chance**. JavaScript functions don't have a fixed arity. You can invoke them with any number of arguments. If you want that to be changed, then you need to beg for a change of the ECMAScript specification ;-)

Reply:
You're right, I didn't think about that